oracle/oracles/local.go

```diff
@@ -51,6 +51,8 @@ type localOracle struct {
 	hook            *struct {
 		currentTime time.Time
 	}
+
+	localExternalTimestamp
```

Comment:
Do we need to set local external ts?

Reply:
The empty value should be valid :thinking: , it only contains an atomic 0.